### PR TITLE
Send port change notifications

### PR DESCRIPTION
**Pull request by pyhys**
Better summary of available diagnosis when creating an alarm

Based on contributions from atomic-3dpenguin

### CODE REVIEW

File: src/common/pf_lldp.c [hunk 1]
```diff
@@ -1020,6 +1020,108 @@ bool pf_lldp_is_peer_info_received (pnet_t * net, uint32_t * timestamp_10ms)
    return is_received;
 }
 
+static void pf_lldp_send_remote_mismatch_alarm (
```
Comment (olbjo) [inline src/common/pf_lldp.c:1023]: I think this operation should be placed in pdport which handles the pdport data checks. Operation name could be "pf_pdport_peer_indication(). Operation called when peer info is changed. Nice if lldp do not need to be aware of alarms and mapping of ports to DAP

File: src/common/pf_lldp.c [hunk 2]
```diff
@@ -1020,6 +1020,108 @@ bool pf_lldp_is_peer_info_received (pnet_t * net, uint32_t * timestamp_10ms)
...
+            lldp_peer_info->chassis_id.string,
+            (char *)p_wanted_peer->peer_station_name,
+            lldp_peer_info->chassis_id.len) == 0)
+      {
+         peer_stationname_is_correct = true;
+      }
+
+      // TODO Update also peer_portname_is_correct
+
+      for (ix = 0; ix < NELEMENTS (net->cmrpc_ar); ix++)
```
Comment (olbjo) [inline src/common/pf_lldp.c:1048]: The observer checking an alarm rule should not need to care whether there is an active ar or not. I think check should be inside alarm, or the loop will be common to all observers and also application/process alarms?

On the other hand what will happen if we do not have an active ar when the check is performed and the lldp frame is received - then we will never send any alarm ( If I do not misunderstand).

**Review by olbjo — CHANGES_REQUESTED** [review]
Lets discuss!

File: src/common/pf_lldp.c (hunk 1 above)
Comment (pyhys) [reply src/common/pf_lldp.c]: Done

**Comment by pyhys** [comment]
Replaced by #246

**Pull request closed without merging**